File: LeetCode-Programs/Consecutive Characters.cpp
class Solution {
public:
    int maxPower(string s) {
    int count=1,maxx=1;
    for(int i=0;i<s.size();i++)
    {
        if(s[i]==s[i+1]) 
            maxx=max(maxx,++count);
       else
           count=1;
    }
    return max(maxx,count);
    }
};

File: LeetCode-Programs/Consecutive Characters.py
class Solution:
    def maxPower(self, s: str) -> int:
        count=1
        maxx=1
        for i in range(1,len(s)):
            if s[i-1]==s[i]:
                count+=1
                maxx=max(maxx, count)
            else:
                count=1
        return max(maxx, count)
